[2023-03-26T00:24:06.821-0400] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_dag.is_api_active manual__2023-03-26T04:17:00.866455+00:00 [queued]>
[2023-03-26T00:24:06.826-0400] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_dag.is_api_active manual__2023-03-26T04:17:00.866455+00:00 [queued]>
[2023-03-26T00:24:06.826-0400] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-03-26T00:24:06.826-0400] {taskinstance.py:1283} INFO - Starting attempt 1 of 1
[2023-03-26T00:24:06.826-0400] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-03-26T00:24:06.834-0400] {taskinstance.py:1303} INFO - Executing <Task(HttpSensor): is_api_active> on 2023-03-26 04:17:00.866455+00:00
[2023-03-26T00:24:06.838-0400] {standard_task_runner.py:55} INFO - Started process 35690 to run task
[2023-03-26T00:24:06.849-0400] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'api_dag', 'is_api_active', 'manual__2023-03-26T04:17:00.866455+00:00', '--job-id', '123', '--raw', '--subdir', 'DAGS_FOLDER/weatherapi_test.py', '--cfg-path', '/var/folders/71/zhs_yc7n05l7c97rnhn92bb80000gn/T/tmpxytrof1m']
[2023-03-26T00:24:06.864-0400] {standard_task_runner.py:83} INFO - Job 123: Subtask is_api_active
[2023-03-26T00:24:07.074-0400] {task_command.py:388} INFO - Running <TaskInstance: api_dag.is_api_active manual__2023-03-26T04:17:00.866455+00:00 [running]> on host sunandas-laptop.hsd1.ga.comcast.net
[2023-03-26T00:24:07.104-0400] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=api_dag
AIRFLOW_CTX_TASK_ID=is_api_active
AIRFLOW_CTX_EXECUTION_DATE=2023-03-26T04:17:00.866455+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-26T04:17:00.866455+00:00
[2023-03-26T00:24:07.104-0400] {http.py:122} INFO - Poking: weather?q=Atlanta,Georgia,US&units=standard&appid=94ca56afa3eee20f7100985fdfe682aa
[2023-03-26T00:24:07.106-0400] {base.py:73} INFO - Using connection ID 'my_api_connection' for task execution.
[2023-03-26T00:25:07.262-0400] {local_task_job.py:276} WARNING - State of this instance has been externally set to queued. Terminating instance.
[2023-03-26T00:25:07.267-0400] {process_utils.py:129} INFO - Sending Signals.SIGTERM to group 35690. PIDs of all processes in the group: [35690]
[2023-03-26T00:25:07.267-0400] {process_utils.py:84} INFO - Sending the signal Signals.SIGTERM to group 35690
[2023-03-26T00:26:07.276-0400] {process_utils.py:147} WARNING - process psutil.Process(pid=35690, name='Python', status='running', started='00:24:06') did not respond to SIGTERM. Trying SIGKILL
[2023-03-26T00:26:07.276-0400] {process_utils.py:84} INFO - Sending the signal Signals.SIGKILL to group 35690
[2023-03-26T00:26:07.281-0400] {process_utils.py:79} INFO - Process psutil.Process(pid=35690, name='Python', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='00:24:06') (35690) terminated with exit code Negsignal.SIGKILL
[2023-03-26T00:26:07.281-0400] {standard_task_runner.py:168} ERROR - Job 123 was killed before it finished (likely due to running out of memory)
